.stylelintrc.json

```diff
@@ -25,7 +25,6 @@
     "scss/no-global-function-names": null,
 
     "scss/dollar-variable-pattern": "^([a-z][a-z0-9]*)((-|--|_|__)[a-z0-9]+)*$",
-    "declaration-colon-newline-after": null,
```

Comment:
how did you know about this? @goapunk

Reply:
the ci failed earlier complaining about this rule,  it's listed here: https://stylelint.io/migration-guide/to-15/ as deprecated and removed in 16.x